src/main/java/ch/njol/skript/Skript.java

```diff
@@ -1470,10 +1471,10 @@ public static <E extends SkriptEvent> SkriptEventInfo<E> registerEvent(String na
 
 		String[] transformedPatterns = new String[patterns.length];
 		for (int i = 0; i < patterns.length; i++)
-			transformedPatterns[i] = "[on] " + SkriptEvent.fixPattern(patterns[i]) + SkriptEventInfo.EVENT_PRIORITY_SYNTAX;
+			transformedPatterns[i] = SkriptEvent.fixPattern(patterns[i]);
 
 		SkriptEventInfo<E> r = new SkriptEventInfo<>(name, transformedPatterns, c, originClassPath, events);
-		structures.add(r);
+		Skript.events.add(r);
```

Comment:
```suggestion
		events.add(r);
```
You're already in the Skript class and registerEvent is static.

Reply:
There's already a parameter called `events` therefore I have to explicitly specify this one

Reply:
We probably shouldn't shadow the `events` field. Maybe call the parameter something like `applicableEvents` or `bukkitEvents`?

Reply:
Why? I don't think there's a problem with it personally

Reply:
It's probably fine in this case, shadowing a member just Feels wrong.

Reply:
Technically under the standard style guide you're not supposed to do that because it introduces ambiguity, but I think it's fine because you're qualifying the field so we know what it is.